<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditOpenPointItemDialog</class>
 <widget class="QDialog" name="EditOpenPointItemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>472</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="subjectLabel">
       <property name="text">
        <string>Subject</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="taskLabel">
       <property name="text">
        <string>Task</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="resultLabel">
       <property name="text">
        <string>Result</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="resultlineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="riskLabel">
       <property name="text">
        <string>Risk</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QComboBox" name="riskcomboBox">
       <item>
        <property name="text">
         <string>Low</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Middle</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>High</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="scheduledfinishLabel">
       <property name="text">
        <string>Scheduled Finish Date</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QDateEdit" name="scheduledfinishdateEdit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="responsibleLabel">
       <property name="text">
        <string>Responsible</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="responsiblelineEdit"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="entrydateLabel">
       <property name="text">
        <string>Entry Date</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QDateEdit" name="entrydatedateEdit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="statusLabel">
       <property name="text">
        <string>Status</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QComboBox" name="statuscomboBox">
       <item>
        <property name="text">
         <string>Work in progress</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Finished</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>On hold</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cancelled</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QTextEdit" name="tasktextEdit">
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="subjectlineEdit">
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>subjectlineEdit</tabstop>
  <tabstop>tasktextEdit</tabstop>
  <tabstop>resultlineEdit</tabstop>
  <tabstop>riskcomboBox</tabstop>
  <tabstop>scheduledfinishdateEdit</tabstop>
  <tabstop>responsiblelineEdit</tabstop>
  <tabstop>entrydatedateEdit</tabstop>
  <tabstop>statuscomboBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditOpenPointItemDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditOpenPointItemDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
